brainiak/fcma/classifier.py

```diff
@@ -43,6 +43,12 @@ class Classifier(BaseEstimator):
     clf: class
         The classifier used, normally a classifier class of sklearn
 
+    num_processed_voxels: int, default 2000
+        Used for SVM with precomputed kernel,
+        every time only compute correlation between num_process_voxels and
+        the whole brain to aggregate the kernel matrices.
+        This is to better use the memory
```

Comment:
yes, computing similarity matrices portion by portion can save the memory so as to handle correlations at a larger scale.